temporalio/client.py

```diff
@@ -2758,6 +2758,12 @@ class WorkflowExecution:
     parent_run_id: Optional[str]
     """Run ID for the parent workflow if this was started as a child."""
 
+    root_id: Optional[str]
+    """ID for the root workflow"""
+
+    root_run_id: Optional[str]
+    """Run ID for the root workflow"""
```

Comment:
```suggestion
    """ID for the root workflow."""

    root_run_id: Optional[str]
    """Run ID for the root workflow."""
```

(pardon the pedantry)

Reply:
no worries :)